platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -5,12 +5,144 @@
 #
 #-------------------------------------------------------------------------------
 
-set(NRF_COMMON_DIR ${CMAKE_CURRENT_LIST_DIR})
+set(NRF_COMMON_DIR "${CMAKE_CURRENT_LIST_DIR}")
 set(NRFX_DIR "${NRF_COMMON_DIR}/nrfx")
+set(PLATFORM_DIR "${NRF_COMMON_DIR}/../../..")
```

Comment:
Is it needed to set it again? It was already set in the top board cmake list.

Reply:
Removed